lib/rucio/core/rule.py

```diff
@@ -818,9 +945,9 @@ def list_rules(filters={}, *, session: "Session"):
             stmt = stmt.where(getattr(models.ReplicationRule, key) == value)
 
     try:
-        for rule, data_identifier_bytes in session.execute(stmt).yield_per(5):
-            d = rule.to_dict()
-            d['bytes'] = data_identifier_bytes
+        for item in session.execute(stmt).yield_per(5):
+            d = item[0].to_dict()
+            d['bytes'] = item[1]
```

Comment:
could you clarify why this change was made?

Reply:
I believe I had to change this when I changed the query statement, but I then reverted the query statement without also reverting this. Reverted the generator back to:

`for rule, data_identifier_bytes in session.execute(stmt).yield_per(5):`